### PR TITLE
Update JMESPath arithmetic functions' docs

### LINKED ISSUE #392: [Docs] JMESPath arithmetic function units
**Related Issue**
- kyverno/kyverno#2500

**Related PR**
- kyverno/kyverno#2753

### PR DESCRIPTION
**Pull request by ojhaarjun1**
Closes #392 

Signed-off-by: Kumar Mallikarjuna <kumarmallikarjuna1@gmail.com>

### CODE REVIEW

File: content/en/docs/Writing policies/variables.md [hunk 1]
```diff
@@ -405,11 +405,21 @@ regex_replace_all(regex string, src string|number, replace string|number) string
...
+add(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
 subtract(number, number) number
+subtract(quantity|number, quantity|number) quantity (returns quantity if any of the parameters is a quantity)
+subtract(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
 multiply(number, number) number
-divide(number, number) number (divisor must be non zero)
-modulo(number, number) number (divisor must be non-zero, arguments must be integers)
+multiply(quantity|number, quantity|number) quantity (returns quantity if any of the parameters is a quantity)
+multiply(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
+divide(quantity|number, quantity|number) quantity|number (returns quantity if exactly of the parameters is a quantity, else a number; the divisor must be non-zero)
```
Comment (chipzoller) [inline content/en/docs/Writing policies/variables.md:418]: "returns quantity if all of the parameters are a quantity, else a number; ..."

File: content/en/docs/Writing policies/variables.md [hunk 2]
```diff
@@ -405,11 +405,21 @@ regex_replace_all(regex string, src string|number, replace string|number) string
...
+divide(duration|number, duration|number) duration|number (returns duration if exactly of the parameters is a duration, else a number; the divisor must be non-zero)
```
Comment (chipzoller) [inline content/en/docs/Writing policies/variables.md:419]: Similar to above comment.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Couple grammar things.

File: content/en/docs/Writing policies/variables.md (hunk 1 above)
Comment (ojhaarjun1) [reply content/en/docs/Writing policies/variables.md:418]: `multiply(number, quantity)` returns a `quantity`

Comment (chipzoller) [reply content/en/docs/Writing policies/variables.md:418]: The comment was for divide.

Comment (ojhaarjun1) [reply content/en/docs/Writing policies/variables.md]: We're returning a quantity for the following cases:
```
divide(number, quantity)
divide(quantity, number)
divide(quantity, quantity)
```

Comment (chipzoller) [reply content/en/docs/Writing policies/variables.md]: First one probably shouldn't return a quantity as that's technically an invalid calculation. But this is a functionality thing and not a docs thing.

Comment (ojhaarjun1) [reply content/en/docs/Writing policies/variables.md]: Yep, doesn't make sense mathematically but we decided to stick to this to keep things simple.

Comment (chipzoller) [reply content/en/docs/Writing policies/variables.md]: So for that are we just assigning the same unit to the number as is present on the quantity? Ex., 10/5Mi is treated as 10Mi/5Mi?

**Comment by chipzoller** [comment]
Can you address any in-line comments and mark as resolved if done, please?

**Comment by chipzoller** [comment]
@ojhaarjun1 ☝️

**Comment by chipzoller** [comment]
Also, conflicts to resolve.

File: content/en/docs/Writing policies/variables.md (hunk 1 above)
Comment (ojhaarjun1) [reply content/en/docs/Writing policies/variables.md]: No, `10/5Mi` is interpreted as `10 Bytes / 5 Mi Bytes` = `2m`

File: content/en/docs/Writing policies/variables.md [hunk 3]
```diff
@@ -406,11 +406,21 @@ regex_replace_all_literal(regex string, src string|number, replace string|number
...
+add(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
 subtract(number, number) number
+subtract(quantity|number, quantity|number) quantity (returns a quantity if any of the parameters is a quantity)
+subtract(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
 multiply(number, number) number
-divide(number, number) number (divisor must be non zero)
-modulo(number, number) number (divisor must be non-zero, arguments must be integers)
+multiply(quantity|number, quantity|number) quantity (returns a quantity if any of the parameters is a quantity)
+multiply(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
+divide(quantity|number, quantity|number) quantity|number (returns a quantity if exactly one of the parameters is a quantity, else a number; the divisor must be non-zero)
```
Comment (chipzoller) [inline content/en/docs/Writing policies/variables.md:419]: Can we put a note here, given recent discussions, that users need to specify inputs in the correct format given the expected input type?

"An input expected to be an integer/number must be wrapped in back ticks while quantity/string should be specified wrapped in single quotes." <== something like this?

**Review by chipzoller — CHANGES_REQUESTED** [review]
See inline comment. Can we add this to help users?

**Comment by ojhaarjun1** [comment]
> See inline comment. Can we add this to help users?

A note seemed more appropriate.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**